src/utils/i18n/enzymeHelper.jsx

```diff
@@ -22,20 +22,21 @@ function nodeWithIntlProp(node) {
   return React.cloneElement(node, { intl });
 }
 
-export function shallowWithIntl(node, { context, disableLifecycleMethods } = {}) {
+export function shallowWithIntl(node, { context, ...otherOptions } = {}) {
   return shallow(
     nodeWithIntlProp(node),
     {
+      ...otherOptions,
       context: Object.assign({}, context, { intl }),
-      disableLifecycleMethods,
     },
   );
 }
 
-export function mountWithIntl(node, { context, childContextTypes } = {}) {
+export function mountWithIntl(node, { context, childContextTypes, ...otherOptions } = {}) {
   return mount(
     nodeWithIntlProp(node),
     {
+      ...otherOptions,
```

Comment:
That looks good to me. Thanks!

Reply:
Do you think it's less clear this way? I'm fine with either option, I think, but at least this would prevent us from having to touch this function as much to add on parameters.

Reply:
I prefer this way. Now it behaves like an actual wrapper around mount/shallow and passes through ALL options, which is what I originally expected.